ci-operator/step-registry/etcd-encryption/vault-install/etcd-encryption-vault-install-commands.sh

```
@@ -0,0 +1,265 @@
+  echo ""
+  echo "Configuring Vault for KMS..."
+
+  # Enable transit secret engine
+  echo "Enabling transit secret engine..."
+  oc exec vault-0 -n "${VAULT_NAMESPACE}" -- \
+    env VAULT_TOKEN="${ROOT_TOKEN}" vault secrets enable -path=transit transit
+
+  # Create encryption key
+  echo "Creating transit encryption key..."
+  oc exec vault-0 -n "${VAULT_NAMESPACE}" -- \
+    env VAULT_TOKEN="${ROOT_TOKEN}" vault write -f transit/keys/${VAULT_KMS_KEY_NAME}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)